docs/source/index.rst

```diff
@@ -58,6 +58,31 @@ the usage notes.
 
 A list of current issues can be found `here <https://github.com/dask/fastparquet/>`_.
 
+Relation to Other Projects
+--------------------------
+
+-  `parquet-cpp <https://github.com/apache/parquet-cpp/>`_ is a low-level C++
+implementation of the parquet format which can be called from python using
+Apache `Arrow <http://pyarrow.readthedocs.io/en/latest/>`_ bindings. We aim to
+collaborate with parquet-cpp, and it is possible, in the medium term, that their low-level
+routines will replace some functions in fastparquet or that high-level logic in
+fastparquet will be migrated to C++.
+
+- `parquet-python <https://github.com/jcrobak/parquet-python>`_ is the original
+pure-python parquet quick-look utility which was the inspiration for fastparquet.
+
+- `pySpark <http://spark.apache.org/docs/2.1.0/programming-guide.html>`_, a python API to the spark
+engine, interfaces python commands with a java/scala execution core, and thereby
+gives python programmers access to the parquet format. fastparquet has no
+defined relationship to pySpark, but can provide an alternative path to providing
+data to Spark or reading data produced by Spark without invoking a pySpark client
+or interacting directly with the Spark scheduler.
+
+- fastparquet lives within the `dask <http://dask.pydata.org>`_ ecosystem, and
+although it is useful by itself, it is designed to work well with dask and
+`distributed <http://distributed.readthedocs.io/en/latest/>`_ for parallel
```

Comment:
Prefer that we don't use the name `distributed` as a proper noun (it's a bad name).  Perhaps just leave it as Dask here.